### ISSUE TITLE
Compatibility issues with Prettier v2.0

### ISSUE THREAD

**Issue by kachkaev** [matzkoh/prettier-plugin-packagejson#17]
[Prettier v2.0 got released](https://prettier.io/blog/2020/03/21/2.0.0.html) 🎉

I can’t wait to try out the new version of prettier this plugin together!  Seems like it needs a bit of tweaking due to this error though.

```
Error: Cannot find module 'prettier/parser-babylon'
Require stack:
- /path/to/project/node_modules/prettier-plugin-packagejson/lib/index.js
- /path/to/project/node_modules/prettier/bin-prettier.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
    at Function.Module._load (internal/modules/cjs/loader.js:686:27)
    at Module.require (internal/modules/cjs/loader.js:848:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/path/to/project/node_modules/prettier-plugin-packagejson/lib/index.js:1:21)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)
    at Module.require (internal/modules/cjs/loader.js:848:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/path/to/project/node_modules/prettier-plugin-packagejson/lib/index.js',
    '/path/to/project/node_modules/prettier/bin-prettier.js'
  ]
}
```

I'd be great if you could help, @matzkoh 🙏

**Comment by laozhu** [comment]
I have the same problem, thank you for any help.

**Comment by kachkaev** [comment]
Competing work-in-progress PRs: #16 & #18

**Comment by matzkoh** [comment]
released v2.1.0 for 2.0.
